Backend/CarRentalApp/CarRentalApp/Services/Token/TokenService.cs

```diff
@@ -38,14 +62,96 @@ public JwtSecurityToken GenerateAccessToken(User user)
             );
         }
 
-        private SigningCredentials GetCredentials(GenerationParameters jwtConfig)
+        public JwtSecurityToken GenerateRefreshToken(User user)
+        {
+            var refreshJwtGenerationParams = _refreshJwtConfig.GenerationParameters;
+
+            var jwtClaims = new List<Claim>()
+            {
+                new Claim(JwtRegisteredClaimNames.Sid, user.Id.ToString()),
+            };
+
+            return new JwtSecurityToken(
+                claims: jwtClaims,
+                expires: DateTime.Now.AddSeconds(refreshJwtGenerationParams.LifeTimeSeconds),
+                signingCredentials: GetCredentials(refreshJwtGenerationParams)
+            );
+        }
+
+        public async Task<RefreshToken?> GetExistingTokenAsync(RefreshTokenDTO tokenDTO)
```

Comment:
Does this name clearly describe what this method responsible for?
When I call Get method I don't expect that it will delete something
You can thing of something like PopExistingTokenAsync